packages/odata-service-inquirer/src/translations/odata-service-inquirer.i18n.json

```diff
@@ -105,7 +105,8 @@
             "newSystemChoiceLabel": "New system",
             "hint": "Select a system configuration.",
             "message": "System",
-            "authenticationFailedUpdateCredentials": "Authentication failed. Please try updating the credentials."
+            "authenticationFailedUpdateCredentials": "Authentication failed. Please try updating the credentials.",
+            "destinationRetrievalError": "An error occurred when retrieving the list of destinations. For more information, view the logs."
```

Comment:
In other messages we state "see logs for further details." , rather than "For more information, view the logs." . There should probably be a "text" entry for that subtext.

Reply:
Thanks @IainSAP , this has been removed anyway. But noted for future